api/jobset/v1alpha2/jobset_webhook.go

```diff
@@ -91,6 +91,11 @@ func (js *JobSet) ValidateCreate() (admission.Warnings, error) {
 		for _, errMessage := range validation.IsDNS1123Subdomain(js.Spec.Network.Subdomain) {
```

Comment:
Is RFC 1035 more emcompassing than DNS 1123?  ie does validating DNS1035 mean that we also satisfy DNS 1123?